x/programs/runtime/dependencies.go

```diff
@@ -25,6 +27,6 @@ type Runtime interface {
 }
 
 type Storage interface {
-	Get(context.Context, uint32) ([]byte, bool, error)
+	Get(context.Context, uint32) (bool, ed25519.PublicKey, []byte, error)
```

Comment:
@hexfusion whats the reason for returning a pubkey?

Reply:
In the context of program to program calls the owner of the program might be useful.

Reply:
But yeah in the context of the interface this isn't ideal, ideally the return object could be generic or maybe a string, this was just wip and need to think about it more.

Reply:
But yeah none of that should be part of the interface the signature should remain `([]byte, bool, error)`